media-querys.css

```diff
@@ -0,0 +1,193 @@
+/* Big screens */
+@media (min-width: 768px) {
+  header.nav {
+    margin-left: 24px;
+  }
+
+  .burguer-img-toggle {
+    display: none;
+  }
+
+  .navbar-menu-mobile {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .navbar-menu-mobile li {
+    padding: 0 15px;
+    align-items: center;
+  }
+
+  .main {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 24px;
+  }
+
+  .text-and-social-wrapper {
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+    align-content: flex-start;
+    justify-content: flex-start;
+    padding-right: 23%;
+    margin: 0 auto;
+  }
+
+  .text-wrapper {
+    grid-area: 1 span 1;
+    order: 1;
+  }
+
+  .social-media {
+    /* display: flex; */
+    grid-area: 0 span 1;
+    flex-direction: column;
+    order: 0;
+  }
+
+  /* works section */
+  .works {
+    width: 90%;
+    margin: auto 24px;
+  }
+
+  .works-title {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .section-works-title {
+    display: flex;
+    border-bottom: none;
+    width: 50%;
+    margin-top: 1.3%;
+    align-items: center;
+  }
+
+  .works-title-div {
+    display: flex;
+    border: #212121 solid 1px;
+    width: 100%;
+    margin: 0 auto;
+    align-self: center;
+  }
+
+  .grid-container {
+    display: grid;
+    max-width: 100%;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+
+    /* margin: 0 24px; */
+
+    /* gap: 24px; */
+  }
+
+  .first-card-flex {
+    display: flex;
+    align-items: center;
+  }
+
+  .card-flex {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .proArt {
+    padding-bottom: 14px;
+  }
+
+  .desktop-display-none {
+    display: none;
+  }
+
+  .card1 {
+    background-image: url("./images/card1.png");
+    padding-bottom: 0;
+  }
+
+  .card1 > .text-container {
+    display: none;
+  }
+
+  .card1 > button {
+    display: block;
+  }
+
+  .card-type1 {
+    background-image: url(./images/card\ type1.png);
+  }
+
+  /* .card-type1 > div > h3 {
+    display: none;
+  } */
+
+  /* .card-type1 > div > .show-on-desktop {
+    display: block;
+  } */
+  .card-type2 {
+    background-image: url(./images/card\ type2.png);
+  }
+
+  /* .card-type2 > div > .show-on-desktop {
+    display: block;
+  } */
+  .card-type3 {
+    background-image: url(./images/card\ type3.png);
+  }
+
+  .hide-on-desktop {
+    display: none;
+  }
+
+  .show-on-desktop {
+    display: block;
+  }
+
+  .text-and-button-wrapper > p {
+    width: 61%;
+  }
+
+  .skills {
+    flex-direction: row;
+    margin: 10% 0;
+    align-items: flex-start;
+  }
+
+  .contact {
+    margin: 0 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
Kindly adjust the contacts section to match the design template. Please, keep to the margins of the section and the paddings of the button.

Your design:
![image](https://user-images.githubusercontent.com/20554583/229231472-6105c01f-065f-4bf3-af50-19bbf9c1453b.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/229231358-63af2f74-08a8-46ac-bc41-b98a22bff054.png)

Reply:
It's done